docs/zh/latest/plugins/file-logger.md

```diff
@@ -143,3 +144,43 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1  \
     }
 }'
 ```
+
+## 配置 matches 参数
```

Comment:
This block should be put before Section Disable Plugin

Reply:
fix~